supabase/migrations/20240712231748_create_Events_table.sql

```diff
@@ -9,5 +9,7 @@ location text,
 timezone text,
 mode text,
 config json,
-creator uuid references Users(id)
+creator uuid references Users(id),
+-- new column to to create a link that can be shared
+shareable_link text
```

Comment:
is shareable link needed?  EventID was used above

Reply:
Removed this.